src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/Image_UseTargetSize.xaml

```diff
@@ -13,7 +13,7 @@
 			<Image Width="100"
 				   Height="100"
 				   Stretch="UniformToFill"
-				   Source="https://lh5.ggpht.com/lxBMauupBiLIpgOgu5apeiX_YStXeHRLK1oneS4NfwwNt7fGDKMP0KpQIMwfjfL9GdHRVEavmg7gOrj5RYC4qwrjh3Y0jCWFDj83jzg" />
```

Comment:
This one needs to be remote, so that loading can actually happen delayed (for all of this file).

Reply:
In that case let's host it on our endpoints though, I saw some of the remote images are hosted on AWS